scripts/docker-utils.sh

```
@@ -0,0 +1,46 @@
+
+  # Check if the container exists before stopping and removing it
+  if docker ps -aq --filter "name=$container_name" | grep -q .; then
+    docker stop "$container_name" >/dev/null 2>&1
+    docker rm "$container_name" >/dev/null 2>&1
+  fi
+}
+
+build_docker_image() {
+  local image_name="$1"
+
+  echo "\nBuilding image"
```

Shellcheck (0.10.0)

[info] echo may not expand escape sequences. Use printf.

(SC2028)